### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by Sspadea**
## Summary ##

**Trello card**: https://trello.com/c/fxVvWh4i/10-n-42-navigation-bar

- Adding navigation bar

## Screenshots ##

Captura de Pantalla 2022-03-11 a la(s) 16 12 09

### CODE REVIEW

File: WBooks/AppDelegate.swift [hunk 1]
```diff
@@ -13,7 +13,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+
```
Comment (NickMano) [inline WBooks/AppDelegate.swift]: You can delete the enters

File: WBooks/LibraryViewController.swift [hunk 2]
```diff
@@ -38,6 +38,29 @@ class LibraryViewController: UITableViewController {
         
         tableView.register(UINib(nibName: cellIdentifier, bundle: nil), forCellReuseIdentifier: cellIdentifier)
         tableView.backgroundColor = UIColor(hex: "#EAF6FA")
+        
+        // NavBar
+        let title = UILabel()
+        title.font = UIFont.boldSystemFont(ofSize: 17)
+        title.textColor = .white
+        title.text = "LIBRARY"
+        //title.frame = CGRect(x: 0, y: 0, width: 71, height: 20)
```
Comment (NickMano) [inline WBooks/LibraryViewController.swift]: Remove commented code

**Review by NickMano — CHANGES_REQUESTED** [review]
Add .DS_Store to git ignore

File: WBooks/LibraryViewController.swift [hunk 3]
```diff
@@ -38,6 +38,26 @@ class LibraryViewController: UITableViewController {
...
+        setNavigationLeftButtons([notifications])
+        setNavigationRightButtons([search])
+
+        let navbarAppearance = UINavigationBarAppearance()
+        navbarAppearance.backgroundColor = UIColor(hex: "#00ADEE")
+
+        navigationController?.navigationBar.isTranslucent = false
+        
+        navigationController?.navigationBar.standardAppearance = navbarAppearance
+        navigationController?.navigationBar.scrollEdgeAppearance = navbarAppearance
```
Comment (dfmm472) [inline WBooks/LibraryViewController.swift]: As a recommendation Split this into a method like setupNavBar

**Review by dfmm472 — APPROVED** [review]

File: WBooks/LibraryViewController.swift (hunk 3 above)
Comment (dfmm472) [reply WBooks/LibraryViewController.swift]: Remove comment code //NavBar

**Review by NickMano — APPROVED** [review]

**Pull request merged**